libjsengine/JSV8Engine.cpp

```diff
@@ -23,6 +23,10 @@
 #include <memory>
 #include "JSV8Engine.h"
 #include "libjsengine/JSEngineResources.hpp"
+#include "BuildInfo.h"
+
+#define TO_STRING_HELPER(s) #s
```

Comment:
I do not want to make libjsengine dependent on libdevcore. On macos `libjsengine` must be compiled with old `libstdc++` instead of `libc++`. I do not want to bring more complexity here.